pkg/didcomm/common/service/service.go

```diff
@@ -32,6 +46,33 @@ type DIDCommMsg struct {
 	OutboundDestination *Destination
 	// ToVerKeys are recipient keys
 	ToVerKeys []string
+	Common    *Common
```

Comment:
Do we need a separate struct for these fields?

Reply:
I think yeah, bc it will not overlap with existing fields (probably somehow in the future), and the main reason was to keep a pointer to determine was there unmarshal already or not. 
<nil> - means unmarshal was not executed